packages/squiggle-lang/src/reducer/lambda.ts

```diff
@@ -182,7 +182,11 @@ export class BuiltinLambda extends BaseLambda {
         .map(fnDefinitionToString)
         .map((def) => `  ${this.name}${def}\n`)
         .join("");
-      return `There are function matches for ${this.name}(), but with different arguments:\n${defsString}`;
+      return `There are function matches for ${
+        this.name
```

Comment:
Can we wrap all parts that look like Squiggle code (or type definitions) in `<code>` or `font-mono` class?

Reply:
We could, but wouldn't that break Terminal compatibility? I assumed you wouldn't want this, for the same reasons you didn't like using Markdown.

Reply:
I didn't realize this is squiggle-lang and not components, sorry. Then this will have to wait until we expose a more structured output. (I don't think it should be a markup that gets parsed on the frontend, though, probably something more like a strictly typed JS object).